uiowa/uiowa
# Feature AoS Search Block removal

Resolves https://github.com/uiowa/uiowa/issues/4082
Resolves https://github.com/uiowa/uiowa/issues/4123





# How to test

`blt ds --site=default`

- Enable areas of study split and `drush @default.local cim`
- Check that content type description mentions area of study block and not page
- Check that `areas_of_study_category` doesn't exist for the content type since we removed permisisons to access this taxonomy

`blt ds --site=admissions.uiowa.edu`

- Verify that new search block is in place on home page and that update hook runs for admissions home page removing any reference to previous search block within revisions
- Verify that old area of study search block has been removed from the areas of study module
